build_tools/scripts/bazel_to_cmake.py

```diff
@@ -363,13 +370,15 @@ def cc_binary(self, **kwargs):
     "deps_block": deps_block,
     }
 
-  def cc_embed_data(self, **kwargs):
-    name_block = self._convert_name_block(**kwargs)
-    srcs_block = self._convert_srcs_block(**kwargs)
-    cc_file_output_block = self._convert_cc_file_output_block(**kwargs)
-    h_file_output_block = self._convert_h_file_output_block(**kwargs)
-    namespace_block = self._convert_cpp_namespace_block(**kwargs)
-    flatten_block = self._convert_flatten_block(**kwargs)
+  def cc_embed_data(self, name, srcs, cc_file_output, h_file_output, cpp_namespace = None, strip_prefix = None, flatten = False, identifier = None, **kwargs):
```

Comment:
format this line?

Reply:
Whoops in merge hell some things go unformatted.